torch/_inductor/codegen/wrapper.py

```diff
@@ -1173,7 +1174,7 @@ def generate_profiler_mark_wrapper_call(self, stack):
             'RECORD_FUNCTION("inductor_wrapper_call", c10::ArrayRef<c10::IValue>({{}}));'
         )
 
-    def codegen_device(self, device):
+    def codegen_option(self, device):
```

Comment:
Can we just fix `codegen_device` without adding the new `codegen_option`?

Reply:
Refactored the code to reuse `codegen_device` in `codegen_option`.
I checked that both of them are needed.